guides/examples/basic-web-app.md

```diff
@@ -162,16 +214,19 @@ ockam policy create --at web --resource tcp-inlet --expression '(= subject.compo
 ockam tcp-inlet create --at /node/web --from 127.0.0.1:$OCKAM_PORT --to /project/default/service/forward_to_db/secure/api/service/outlet
 ```
 
-Take note of the `--from` and `--to` values above.
+Please take note of the `--from` and `--to` values above.
 
-* The `--from` is telling the node to listen on port `$OCKAM_PORT` (`5433`).
+* The `--from` tells the node to listen on port `$OCKAM_PORT` (`5433`).
 * And to forward it `--to` the relay service to the database that we created in the previous section.
 * This means requests to `localhost:5433` will be forwarded to whatever node has registered as `db`, wherever it is!
 
 One more change needs to be made to `main.py`. We have to update it to use the new `$OCKAM_PORT` inlet (line 12).
 
 {% hint style="info" %}
-In production environments you would typically get this from an environment variable which wouldn't require code changes. You would have to change the machine configuration running the web app to have the new value of `$OCKAM_PORT`. However, in this example we will just update line 12 in the `main.py` itself to store the new value.
+In production environments you would typically get this from an environment variable which
```

Comment:
I think that we can dispense with this explanation altogether and in the Python application write:
```python
pg_port = os.environ['OCKAM_PORT']
```